app/src/main/java/com/lastaoutdoor/lasta/ui/screen/profile/ProfileScreen.kt

```diff
@@ -429,89 +435,99 @@ fun Chart(activities: List<UserActivity>, timeFrame: TimeFrame, sport: ActivityT
 fun RecentActivities(
     activities: List<UserActivity>,
 ) {
-  Text("Recent Activities", style = TextStyle(fontSize = 20.sp), fontWeight = FontWeight.Bold)
-  for (a in activities.reversed()) {
-    val sport = a.activityType
-    Card(
-        modifier = Modifier.padding(12.dp).fillMaxWidth().testTag("RecentActivitiesItem"),
-        elevation = CardDefaults.cardElevation(4.dp),
-        shape = RoundedCornerShape(8.dp)) {
-          Row(
-              modifier = Modifier.padding(8.dp, 0.dp),
-              verticalAlignment = Alignment.CenterVertically) {
-                // Image on the left
-                Box(modifier = Modifier.size(100.dp).padding(8.dp)) {
-                  /*
-                  Image(
-                      bitmap = imageBitmap,
-                      contentDescription = "Trail Image",
-                      contentScale = ContentScale.Crop,
-                  )
-                   */
-                }
-
-                // Spacer between the image and the text
-                Spacer(modifier = Modifier.width(8.dp))
 
-                Column { Text(text = formatDate(a.timeStarted), fontWeight = FontWeight.Bold) }
-              }
-          // Text information on the right
-
-          Row(
-              modifier = Modifier.fillMaxWidth().padding(16.dp),
-              horizontalArrangement = Arrangement.SpaceBetween) {
-                Column {
-                  when (sport) {
-                    ActivityType.HIKING -> {
-                      val trail = a as HikingUserActivity
-                      Text(
-                          text =
-                              String.format(
-                                  "%.2f", metersToKilometers(trail.distanceDone.toLong())),
-                          fontWeight = FontWeight.Bold)
-                      Text(text = "Km")
-                    }
-                    ActivityType.CLIMBING -> {
-                      val climb = a as ClimbingUserActivity
-                      Text(
-                          text =
-                              String.format(
-                                  "%.2f", metersToKilometers(climb.totalElevation.toLong())),
-                          fontWeight = FontWeight.Bold,
-                          modifier = Modifier.testTag("TestAndrew1"))
-                      Text(text = LocalContext.current.getString(R.string.elevation))
-                    }
-                    ActivityType.BIKING -> {}
+  Text(
+      "Recent Activities",
```

Comment:
Should have this in string resources

Reply:
Thank you for pointing it out, indeed I didn't see it ! Made the change rn